frontend/src/pages/MapView.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
```

Comment:
@asanchezr I've updated the mapview with some example components that display the new redux-based API data. These are just for reference, and are not intended to survive this sprint.